src/components/dar_collection_table/CollectionConfirmationModal.js

```diff
@@ -7,7 +7,7 @@ import {getProjectTitle} from './DarCollectionTable';
 export default function CollectionConfirmationModal(props) {
   const {collection, showConfirmation, setShowConfirmation, cancelCollection, reviseCollection, openCollection, consoleAction, deleteDraft} = props;
 
-  const getModalHeader = () => {
+  const getModalHeader = (collection) => {
     if(!isNil(collection)) {
       return `${collection.darCode} - ${getProjectTitle(collection)}`;
```

Comment:
Optional nit: I don't see `getProjectTitle` used anywhere but here, so maybe inline `collection.name` here and delete the function